tucson-teststand/MT/docker-compose.yaml

```diff
@@ -0,0 +1,52 @@
+version: "3.6"
+
+services:
+
+  mtm1m3-sim:
+    image: ts-dockerhub.lsst.org/mtm1m3_sim:${CYCLE}
+    container_name: mtm1m3-sim
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
+      - OSPL_MASTER_PRIORITY=0
+      - OSPL_INFOFILE=/tmp/ospl-info-mtm1m3.log
+      - OSPL_ERRORFILE=/tmp/ospl-error-mtm1m3.log
+    volumes:
+      - ${OSPL_SHMEM_CONFIG}:${OSPL_MOUNT_POINT}
+      - ${OSPL_SHMEM_SOCKET_DIR}:/tmp/
+    tty: true
+    ipc: host
+    pid: host
+    network_mode: host
```

Comment:
Ok, I noticed you are starting a daemon when you start m1m3. Is this how you plan to deploy this component? starting a daemon at run time? If so, then yes, keep the host network. If not, then you may want to remove this. If you don't know, you probably want to keep it then.

Reply:
Startup script is still there but the docker-compose spec doesn't use it. This will look like how we deploy on NTS now.